src/ecc/curves.iced

```diff
@@ -208,6 +219,18 @@ exports.Curve25519 = class Curve25519 extends Curve
 
   #----------------------------------
 
+  # Serialize single curve coordinate (usually private key X) to MPI
+  # buffer, with 2-byte header encoding bit-size of the coordinate.
+  coord_to_mpi_buffer : (p) ->
+    byte_size = @mpi_coord_byte_size()
+    throw new Error "Invalid p.length: got #{p.length} expected #{byte_size}." if p.length != byte_size
```

Comment:
right but an uncatched error here is a bad thing, it might crash the running program and can be a DoS vector